### PR TITLE
LPS-97199 Add AC experiment on after create DXP experiment

### PR DESCRIPTION
**Pull request by epgarcia**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@darquesdev can you resend this? In general, @Components are not free.. they slow down OSGi startup so we want to conserve.

For ExperimentDTOConverter, make it like

modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-impl/src/main/java/com/liferay/headless/admin/taxonomy/internal/dto/v1_0/util/CreatorUtil.java

In general, whenever I see @Component, I ask myself, is this an internal only class? If so, I try to find the root usage, and then pass the dependencies so there is less to wire.

Basically, wire all the stuff inside SegmentsExperimentModelListener

and then pass it as vars to the other 2 components. If the components are reused, go ahead and do what you did and just pay for the component. But since I only see one usage, let's not pay for it.

@epgarcia fyi

**Comment by darquesdev** [comment]
Great advice!  Moreover,  doing this the code will be more testable.  I'll resend next day. Thanks

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
